CVE-2021-22205/ubuntu1.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Install CVE-2021-22205 PoC for GitLab Pre-Auth Remote Command Execution
+  hosts: all
+  become: yes
+  tasks:
+    - name: Update apt package index
+      apt:
+        update_cache: yes
+
+
+    - name: Install software-properties-common
+      apt:
+        name: software-properties-common
+        state: present
+
+    - name: Add deadsnakes PPA
+      apt_repository:
+        repo: ppa:deadsnakes/ppa
+        state: present
+
+
+    - name: Ensure Docker is installed
+      apt:
+        name: docker.io
+        state: present
+
+    - name: Install a list of packages
+      ansible.builtin.apt:
+        pkg:
+        - docker-buildx
+        - docker-compose-v2
+        
+    - name: Ensure Docker service is started and enabled
+      service:
+        name: docker
+        state: started
+        enabled: yes
+
+    - name: Verify Docker Compose installation
+      command: docker compose version
```

Comment:
Please use built-in Docker commands: https://docs.ansible.com/ansible/latest/collections/community/docker/index.html